frontend/src/app/testResults/viewResults/actionMenuModals/TestResultDetailsModal.tsx

```diff
@@ -204,26 +183,40 @@ export const DetachedTestResultDetailsModal = ({
             removed={removed}
             aria-describedby="result-detail-title"
           />
-          <DetailsRow
-            label="Symptoms"
-            value={
-              symptomList.length > 0 ? symptomList.join(", ") : "No symptoms"
-            }
-            removed={removed}
-            aria-describedby="result-detail-title"
-          />
-          <DetailsRow
-            label="Symptom onset"
-            value={symptomOnset && formatDateWithTimeOption(symptomOnset)}
-            removed={removed}
-            aria-describedby="result-detail-title"
-          />
+          {!isHIVResult && (
```

Comment:
⚠️ We are currently not collecting this for HIV results so symptoms and symptom onset date will only be displayed if it is not a HIV result